### ISSUE TITLE
Support GH Actions badges

### ISSUE THREAD

**Issue by zoispag** [RocktimSaikia/badge-it#14]
I understand the issue with the action right now, as discussed in #11 .
Once this issue is resolved, are there any plans to support badges for GitHub actions' builds?

**Comment by rocktimsaikia** [comment]
Yes. Most definitely. I had plans to do it before but didn't add as this issue popped up. Also due to lack of time because of my internship , I am not able to look deep into issue #11 . So I added some labels there just in case anyone wants to work on that in the meantime, If not then I will look into that personally in near future.

**Comment by zoispag** [comment]
That's good to hear! And the lack of time is totally understandable! I might take a look into #11 if time permits!
